crates/symbolicator-service/src/services/symbolication/sourcemap.rs

```diff
@@ -172,9 +172,57 @@ fn fold_function_name(function_name: &str) -> String {
     format!("{folded}.{tail}")
 }
 
+/// Trims a line down to a goal of 140 characters, with a little wiggle room to be sensible
+/// and tries to trim around the given `column`. So it tries to extract 60 characters
+/// before and after the provided `column` and yield a better context.
+fn trim_context_line(line: &str, column: Option<usize>) -> String {
+    use std::cmp::{max, min};
+
+    let mut line = line.trim_end_matches('\n').to_string();
+    let len = line.len();
+
+    if len <= 150 {
+        return line;
+    }
+
+    let col = column.map(|c| min(c, len)).unwrap_or_default();
+
+    let mut start = max(col.saturating_sub(60), 0);
+    // Round down if it brings us close to the edge.
+    if start < 5 {
+        start = 0;
+    }
+
+    let mut end = min(start + 140, len);
+    // Round up to the end if it's close.
+    if end > len.saturating_sub(5) {
+        end = len;
+    }
+
+    // If we are bumped all the way to the end, make sure we still get a full 140 chars in the line.
+    if end == len {
+        start = max(end.saturating_sub(140), 0);
```

Comment:
```suggestion
        start = end.saturating_sub(140);
```
Same here, and it will indeed saturate to `0`.